modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -284,17 +293,10 @@ export default {
       return this.filterOutParkedFields(fields);
     },
     modalTitle() {
-      if (this.docId) {
-        return {
-          key: 'apostrophe:editType',
-          type: this.$t(this.moduleOptions.label)
-        };
-      } else {
-        return {
-          key: 'apostrophe:newDocType',
-          type: this.$t(this.moduleOptions.label)
-        };
-      }
+      return {
```

Comment:
We now have the same title when we create a new piece and edit an existing document?

Reply:
Hum nope the opposite, if there is a `currentId` means we're editing otherwise creating.